flake.nix

```diff
@@ -182,7 +182,7 @@
 
             # We use the ones from Nixpkgs, since they are cached reliably.
             # Eventually we will probably want to build these with haskell.nix.
-            nativeBuildInputs = [ pkgs'.cabal-install pkgs'.hlint pkgs'.haskellPackages.fourmolu ];
+            nativeBuildInputs = [ pkgs.cabal-install pkgs.hlint pkgs.haskellPackages.fourmolu pkgs.haskellPackages.cabal-fmt pkgs.nixpkgs-fmt ];
```

Comment:
This is wrong FWIW. The `'` have meaning.

Reply:
Yes, I missed it during the rebase. I have fixed this.